### PR TITLE
feat(empty-stacktraces): Tag events with stack traces from JS console errors

### PR DESCRIPTION
**Pull request by snigdhas**
Add a new tag, empty_stacktrace.js_console, to tag JavaScript console errors.

This tag will available to filter issues on the "For Review" tab, based on a solution here https://github.com/getsentry/sentry-javascript/issues/5179.

[Spec](https://www.notion.so/sentry/Filtering-out-JS-console-errors-f9401214cc1f46a6a7dbd05740d1c805)
WOR-2202

### CODE REVIEW

**Review by armenzg — APPROVED** [review]
I have compared it with #39335 and it fixes the issues from last time.

File: src/sentry/lang/javascript/processor.py [hunk 1]
```diff
@@ -1194,8 +1196,38 @@ def process_frame(self, processable_frame, processing_task):
...
+            try:
+                set_tag(self.data, "empty_stacktrace.js_console", suspected_console_errors)
+            except Exception as exc:
+                logger.error(
+                    "Failed to tag event with empty_stacktrace.js_console=%s for suspected JavaScript browser console error",
+                    suspected_console_errors,
+                    exc_info=exc,
+                )
+
+        try:
```
Comment (snigdhas) [inline src/sentry/lang/javascript/processor.py:1223]: Moved the flag check into a try/except block

File: tests/sentry/lang/javascript/test_processor.py [hunk 2]
```diff
@@ -94,6 +97,110 @@ def test_missing_dist(self, _1, _2):
         assert processor.dist.name == "foo"
         assert processor.dist.date_added.timestamp() == processor.data["timestamp"]
 
+    @with_feature("organizations:javascript-console-error-tag")
```
Comment (snigdhas) [inline tests/sentry/lang/javascript/test_processor.py:100]: Restructured code to add the flag to the tests

File: src/sentry/lang/javascript/processor.py [hunk 3]
```diff
@@ -1194,8 +1196,38 @@ def process_frame(self, processable_frame, processing_task):
...
+            if features.has(
+                "organizations:javascript-console-error-tag", self.organization, actor=None
```
Comment (snigdhas) [inline src/sentry/lang/javascript/processor.py:1225]: Fixed the parameter to be `self.organization`.

**Review by snigdhas — COMMENTED** [review]
I'll merge this tomorrow morning so we can keep a closer eye on the deploy.

**Pull request merged**